docs/src/main/sphinx/connector/memsql.rst

```diff
@@ -106,5 +106,8 @@ statements, the connector supports the following features:
 * :doc:`/sql/drop-table`
 * :doc:`/sql/create-schema`
 * :doc:`/sql/drop-schema`
+* :doc:`/sql/alter-schema`
 
 .. include:: sql-delete-limitation.fragment
+
+.. include:: alter-schema-rename.fragment
```

Comment:
Maybe `alter-schema-limitation.fragment` is a better name? Feel free to ignore if you find the current one better.

Reply:
Nah, I think you're right.